tests/queries/0_stateless/02888_parallel_view_query_execution.sql

```diff
@@ -0,0 +1,32 @@
+DROP TABLE IF EXISTS src_02888;
+DROP TABLE IF EXISTS mv_02888;
+
+CREATE TABLE src_02888 (id UInt8) ENGINE = Null;
+CREATE MATERIALIZED VIEW mv_02888 ENGINE = Memory AS SELECT id FROM src_02888 WHERE id % 2 = 0;
+
+SET parallel_view_query_execution = 1;
+SET log_queries = 1;
+SET log_query_threads = 1;
+INSERT INTO src_02888 VALUES (1), (2);
+SET log_queries = 0;
+SET log_query_threads = 0;
+SYSTEM FLUSH LOGS;
+
+SELECT COUNT() > 1 from system.query_thread_log WHERE query LIKE 'INSERT INTO src_02888 VALUES%' AND current_database = currentDatabase();
+
+TRUNCATE system.query_thread_log;
+
+SET parallel_view_query_execution = 0;
+SET log_queries = 1;
+SET log_query_threads = 1;
+INSERT INTO src_02888 VALUES (3), (4);
+SET log_queries = 0;
+SET log_query_threads = 0;
+SYSTEM FLUSH LOGS;
+
+SELECT COUNT() > 1 from system.query_thread_log WHERE query LIKE 'INSERT INTO src_02888 VALUES%' AND current_database = currentDatabase();
```

Comment:
Test looks wrong, it does not checks that in case of `parallel_view_query_execution=0` there are less threads then in case of `parallel_view_query_execution=1`.

Reply:
When parallel_view_query_execution=0, there must be strictly one thread, and SELECT COUNT() > 1 will return 0, which is required in the reference file. On the other hand, when parallel_view_query_execution=1, there must be more than one thread, and SELECT COUNT() > 1 will return 1, which is also required in the reference file. So, the condition you mentioned will be met.

Reply:
Oh, I completely forgot to look at the reference file. Sorry for the noise.